thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -54,6 +54,11 @@ def after_invocation(self, data):
             self.invocation_data['errorType'] = error_type.__name__
             self.invocation_data['errorMessage'] = str(error)
 
+        timeout_exists = False
+        if 'timeout' in data:
+            timeout_exists = data['timeout']
+        self.invocation_data['timeout'] = timeout_exists
```

Comment:
if data here is a dict you can simply use this function:

http://www.tutorialspoint.com/python/dictionary_get.htm